platformio.ini

```diff
@@ -97,10 +98,66 @@ build_flags =
 	-DDISABLE_ALL_LIBRARY_WARNINGS
 	-DDISABLE_SERIAL_TIMEOUT
 	; -DCONFIG_NIMBLE_CPP_LOG_LEVEL=2 ; more verbose BLE logging
+	-DTTGO
 board_build.partitions = default_8MB.csv
 board_build.arduino.memory_type = qio_opi
 board_build.flash_size = 8MB
 lib_deps = 
 	h2zero/NimBLE-Arduino@^1.4.0
 	ottowinter/ESPAsyncWebServer-esphome@^3.0.0
-lib_extra_dirs = ttgo_s3_libs	; needs custom drivers for parallel interface
\ No newline at end of file
+lib_extra_dirs = ttgo_s3_libs	; needs custom drivers for parallel interface
+
+[env:esp32-2432S028R]
```

Comment:
Can you come up with a simpler name for this board here? Will make it easier for people to talk about and reason about if we add it to the install page.

Reply:
Any good idea? This is the link where I bought it: https://de.aliexpress.com/item/1005004502250619.html 
They don't use a brand name or something similar. This "string" is how you easily find it. There are other variations that include 3.5 inch displays and also capacitive touchscreens. So somehow we still need to differenciate them because they probably will need different configurations.

Reply:
What about `JCZN-28R`, admit it's not *much* better but at least it isn't just a string of numbers?

Reply:
done :-)